godtools/App/Features/Tutorial/Presentation/Tutorial/TutorialView.swift

```diff
@@ -7,10 +7,9 @@
 //
 
 import UIKit
-import youtube_ios_player_helper
 
+// TODO: Look into using TutorialPagerView instead. ~Levi
```

Comment:
Would you like to do this in another PR?  I'm hoping it won't be too difficult to apply TutorialPagerView here.

Reply:
Hey @reldredge71 I'm going to create a separate Jira Task for that.  There is another place that needs the TutorialPageView as well which can be part of that task.